frontend/src/locales/en/translation.json

```diff
@@ -896,7 +896,14 @@
     "ToggleSummary": "Show description below image",
     "Back": "Back",
     "DisabledToolTip": "Upload a file to continue",
-    "Cancel": "Cancel"
+    "Cancel": "Cancel",
+    "ImageSize": "Image size (optional)",
+    "ImageSizeHint": "How your image will display in the dashboard.",
+    "AsUploaded": "As uploaded (do not scale)",
+    "100": "100%",
```

Comment:
suggestion: use interpolation

```
"percentage": "{percent}%"
```

and use

```
t("percentage", { percent: scalePct })
```

Reply:
this percentage can be global and you can reuse across the app. x% have no changing context